---
title: Longitudinal sequencing reveals polygenic and epistatic nature of genomic response to selection
author:
  - Simon K.G. Forsberg:
      institute:
        - lsi
        - eeb
        - uppsala
      equal_contributor: "yes"
  - Diogo Melo:
      institute:
        - lsi
        - eeb
      email: damelo@princeton.edu
      orcid: 0000-0002-7603-0092
      equal_contributor: "yes"
  - Scott Wolf:
      institute:
        - lsi
        - eeb
      email: swwolf@princeton.edu
      orcid: 0000-0003-4397-1395
  - Jennifer K. Grenier:
      institute: cornell
  - Minjia Tang:
      institute:
        - lsi
        - eeb
  - Lucas P. Henry:
      institute:
        - lsi
        - eeb
  - Luisa F. Pallares:
      institute: fml
      orcid: 0000-0001-6547-1901
  - Andrew G. Clark:
      institute:
        - cornell
  - Julien Ayroles:
      institute:
        - lsi
        - eeb
      email: jayroles@princeton.edu
      correspondence: "yes"
      orcid: 0000-0001-8729-0511
institute:
  - lsi:
      name: Lewis-Sigler Institute for Integrative Genomics, Princeton University
  - eeb:
      name: Department of Ecology and Evolutionary Biology, Princeton University
  - fml:
      name: Friedrich Miescher Laboratory, Max Planck Society
  - cornell:
      name: Department of Molecular Biology and Genetics, Cornell University, Ithaca, NY, USA
  - uppsala:
      name: Department of Medical Biochemistry and Microbiology, Uppsala University, Sweden.

output: pdf_document
geometry:
- top=20mm
- left=25mm
- right=25mm
- bottom=20mm
header-includes:
- \usepackage[backref=true,style=authoryear]{biblatex}
- \DefineBibliographyStrings{english}{backrefpage = {page}, backrefpages = {pages}}
- \usepackage{multicol}
- \usepackage{setspace}
- \usepackage{float}
- \usepackage{afterpage}
- \usepackage{stfloats}
- \usepackage{graphicx}
- \newcommand{\hideFromPandoc}[1]{#1}
- \hideFromPandoc{ \let\Begin\begin \let\End\end}
link-citations: yes
mainfont: Skolar PE TEST Regular
mainfontoptions:
- Numbers=Lowercase
- Numbers=Proportional
csl: ./pandoc/apa.csl
sansfont: Skolar Sans PE TEST
bibliography: ./references.bib
keywords: [E&R, polygenic selection, Drosophila, High-Sugar Stress, soft sweep]
---

# Introduction

Changes underlying evolutionary response to a new environment can differ depending on the genetic architecture of the traits under selection. For traits with simple genetic architectures, controlled by few genes, we would expect to see selective sweeps, in which a positively selected allele rapidly goes to fixation and leaves a detectable signature in the surrounding genomic regions [@Smith1974-jb; @Pavlidis2017-dd]. Interest in finding the causal allele responsible for a phenotype has caused a bias for this type of selective response in the literature [@Pritchard2010-rr], and we have cataloged several positively selected genes in humans [@Akey2009-pp] and other species [@Ihle2006-dc; @Garud2015-iq; @Feder2016-io]. In contrast, for traits that are controlled by many genes, quantitative genetic theory predicts that selection response is generated by more subtle changes across the allele frequency of many loci [@Jain2017-zd; @Chevin2008-gx], and indeed many have argued that this is a much more prevalent form of selection response than sweeps [@Pritchard2010-va; @Rockman2012-lz; @Barghi2020-ie]. If much of the selection response for complex traits is due to polygenic shifts rather than selective sweeps, the long list of sweep-like selection signatures that have been reported to date might represent the exception rather than the rule.

There is more to genetic architecture than the number of loci affecting a trait [@Hansen2006-ka]. Gene by gene and gene by environment interactions both contribute to genetic variation, and can be leveraged in response to selection. The extend to which epistasis is important for polygenic response is still an open question [@Crow2010-fm;  @Hansen2013-uj; @Phillips2008-ru], but more recent focus on several traits and multivariate responses to selection has provided a large number of possible ways for epistatic interaction to contribute to evolutionary response [@Wolf2005-nr; @Pavlicev2010-po]. Given the possibility that epistasis contributes appreciably to adaptation, we argue that under fitness epistasis, polygenic selection response should result in two observable patterns: 1) a correlation between the allele frequencies at interacting loci - changes in allele frequency at one locus is accompanied by corresponding changes at the (unlinked) interacting locus; 2) the emergence of gametic disequilibrium in adapted populations, as allelic combinations are selected for or against, resulting in deviations from Mendelian proportions between pairs of unlinked loci [@Boyrie2021-jo].

Identifying the signatures of polygenic and epistatic response to selection is a challenging problem, and Evolve and Resequence (E&R) experiments have emerged as a natural and powerful tool for investigating these questions [@Barghi2020-ie]. 
By exposing populations to a stressful treatment condition and keeping them in this environment for several generations, without any sort of phenotype tracking or artificial selection, we are effectively letting natural selection take over and tracking the resulting changes in genetic composition of the selected populations. One consequence of this is that the flies are also under natural selection due to laboratory conditions, which introduces selective pressures other than the experimental treatment. Some E&R experiments remedy this by starting with laboratory adapted populations [eg., @Turner2011-tc], but these populations could still be changing in response to lab conditions even after many generations, and even subtle changes in rearing due to the selection experiment conditions could lead to confounds. We approach this problem by tracking the allele frequency changes in the selected and control populations at several generations, not just at the beginning and at the end of the experiment. This allows for the separation of the genomic effects of the stressor from the effects of the general laboratory environment.


Here, we performed an E&R experiment where three replicate populations of _Drosophila melanogaster_ were exposed to a stressful environment in the form of a high level of dietary sugar, while another three replicate populations were maintained on a control diet. All six populations were derived from the same base population. Whole genome sequencing was performed on flies from all six populations at generations 1, 11, 25, and 100, giving a total of almost 3000 sequenced individuals. Using this time series genomic data, we identify two directions of allele frequency change. The largest driver was shared across selected and control populations, suggesting a shared lab environment selection. The second largest driver of genetic change contrasts control and selected populations, and so is linked to the selection regime. We estimate that at least 6% of the genome was under positive selection due to high sugar stress. Most observed changes in allele frequency are however relatively modest, and using the individual sequence data to estimate haplotypes, we show how most of the selected loci do not show archetypal signals of selective sweeps after 100 generations. These results point towards a highly polygenic selection response, in line with theoretical expectations from quantitative genetics theory. We also quantify correlations in allele frequency between pairs of selected loci over time, as well as gametic disequilibrium after 100 generations of adaptation, and show that several allele show correlations and gametic disequilibrium across unlinked loci, suggesting that epistatic interactions participated in the response to selection. We confirm this last point by using Wright-Fisher simulations, showing that the correlations and the gametic disequilibrium we observe are unlikely to appear in the absence of epistatic interactions. While we lack a clear phenotype to directly study the effect of epistatic variation on selection response, our results suggest that epistatic contributions to polygenic response to selection are common and lead to detectable genomic signatures.

# Methods


# Results

## Polygenic Selection Response

After quality control (see Methods), we obtained allele frequency estimates for ~1.76M SNPs, giving a total of 4 time points x 3 replicate populations x 2 treatments x 1.76M SNPs data points. To identify the main drivers of genetic change without any prior assumptions, we performed a Principal Component Analysis (PCA) of the allele frequencies across the entire selection experiment. The first two principal components (PC), explaining 17% and 13% of the variance respectively, largely coincides with time and selection regime ([@fig:pca; @fig:PCscores_pvals]). This unsupervised approach thus identified time and high sugar selection as the two main drivers of genetic change genome wide. Surprisingly, the time dimension, captured by the first PC, explained slightly more variance than the selection regime, which was captured by the second PC. This shows that all six populations experience some common selection pressures, presumably related to lab environment. The second largest driver of genetic change is indeed exposure to high sugar stress ([@fig:pca]). 

![(A) Principal components one (x-axis) and two (y-axis) from the PCA on the genome wide allele frequencies across the entire selection experiment. Each line corresponds to one of the six experimental populations, red indicating high sugar treatment and blue control. The symbols indicate the generation at which the population was sampled. The variance explained per principal component is shown in the inset. (B) Manhattan plot showing negative log10 transformed p-values from the regression analysis of allele frequency over time. The p-values correspond to the time-by-selection regime interaction coefficient in the model. A significant p-value thus indicates different trajectories in the two treatments. SNPs showing a selection response primarily in the control regime are not shown. ](./figures/PCs12_inset_allSNPs.png){ #fig:pca }

# Discussion

Clear polygenic response, in several replicas. Clear albeit indirect evidence of epistatic interactions in the responding loci.

\footnotesize

## Supporting information

Supporting information can be found at [github.com/diogro/HighSugarSelection](https://github.com/diogro/HighSugarSelection).

### Code and data availability

Code to reproduce figures in this paper is available at [github.com/ayroles-lab/highsugar-selection-code](https://github.com/ayroles-lab/highsugar-selection-code).

Corresponding data is available at **Princeton DataSpace link**.

# References





![Comparison of PC scores and p-values from regression analysis. PC1 scores are similar to p-values from SNPs that show a shared selection signal across treatment and control, and PC2 scores are similar to p-values from SNPs that only show selection signal in HS.](./figures/placeholder.jpg){ #fig:PCscores_pvals }