---
title: Longitudinal sequencing reveals polygenic and epistatic nature of genomic response to selection
author:
  - Simon K.G. Forsberg:
      institute:
        - lsi
        - eeb
        - uppsala
      equal_contributor: "yes"
  - Diogo Melo:
      institute:
        - lsi
        - eeb
      email: damelo@princeton.edu
      orcid: 0000-0002-7603-0092
      equal_contributor: "yes"
  - Scott Wolf:
      institute:
        - lsi
        - eeb
      email: swwolf@princeton.edu
      orcid: 0000-0003-4397-1395
  - Jennifer K. Grenier:
      institute: cornell
  - Minjia Tang:
      institute:
        - lsi
        - eeb
  - Lucas P. Henry:
      institute:
        - lsi
        - eeb
  - Luisa F. Pallares:
      institute: fml
      orcid: 0000-0001-6547-1901
  - Andrew G. Clark:
      institute:
        - cornell
  - Julien Ayroles:
      institute:
        - lsi
        - eeb
      email: jayroles@princeton.edu
      correspondence: "yes"
      orcid: 0000-0001-8729-0511
institute:
  - lsi:
      name: Lewis-Sigler Institute for Integrative Genomics, Princeton University
  - eeb:
      name: Department of Ecology and Evolutionary Biology, Princeton University
  - fml:
      name: Friedrich Miescher Laboratory, Max Planck Society
  - cornell:
      name: Department of Molecular Biology and Genetics, Cornell University, Ithaca, NY, USA
  - uppsala:
      name: Department of Medical Biochemistry and Microbiology, Uppsala University, Sweden.

output: pdf_document
geometry:
- top=20mm
- left=25mm
- right=25mm
- bottom=20mm
header-includes:
- \usepackage[backref=true,style=authoryear]{biblatex}
- \DefineBibliographyStrings{english}{backrefpage = {page}, backrefpages = {pages}}
- \usepackage{multicol}
- \usepackage{setspace}
- \usepackage{float}
- \usepackage{afterpage}
- \usepackage{stfloats}
- \usepackage{graphicx}
- \usepackage{wasysym}
- \usepackage{marvosym}
- \newcommand{\hideFromPandoc}[1]{#1}
- \hideFromPandoc{ \let\Begin\begin \let\End\end}
link-citations: yes
mainfont: Skolar PE
mainfontoptions:
- Numbers=Lowercase
- Numbers=Proportional
csl: ./pandoc/apa.csl
sansfont: Skolar Sans PE
bibliography: ./references.bib
keywords: [E&R, polygenic selection, Drosophila, High-Sugar Stress, soft sweep]
---

# Introduction

Changes underlying evolutionary response to a new environment can differ depending on the genetic architecture of the traits under selection. For traits with simple genetic architectures, controlled by few genes, we would expect to see selective sweeps, in which a positively selected allele rapidly goes to fixation and leaves a detectable signature in the surrounding genomic regions [@Smith1974-jb; @Pavlidis2017-dd]. Interest in finding the causal allele responsible for a phenotype has caused a bias for this type of selective response in the literature [@Pritchard2010-rr], and we have cataloged several positively selected genes in humans [@Akey2009-pp] and other species [@Ihle2006-dc; @Garud2015-iq; @Feder2016-io]. In contrast, for traits that are controlled by many genes, quantitative genetic theory predicts that selection response is generated by more subtle changes across the allele frequency of many loci [@Jain2017-zd; @Chevin2008-gx], and indeed many have argued that this is a much more prevalent form of selection response than sweeps [@Pritchard2010-va; @Rockman2012-lz; @Barghi2020-ie]. If much of the selection response for complex traits is due to polygenic shifts rather than selective sweeps, the long list of sweep-like selection signatures that have been reported to date might represent the exception rather than the rule.

There is more to genetic architecture than the number of loci affecting a trait [@Hansen2006-ka]. Gene by gene and gene by environment interactions both contribute to genetic variation, and can be leveraged in response to selection. The extend to which epistasis is important for polygenic response is still an open question [@Crow2010-fm;  @Hansen2013-uj; @Phillips2008-ru], but more recent focus on several traits and multivariate responses to selection has provided a large number of possible ways for epistatic interaction to contribute to evolutionary response [@Wolf2005-nr; @Pavlicev2010-po]. Given the possibility that epistasis contributes appreciably to adaptation, we argue that under fitness epistasis, polygenic selection response should result in two observable patterns: 1) a correlation between the allele frequencies at interacting loci - changes in allele frequency at one locus is accompanied by corresponding changes at the (unlinked) interacting locus; 2) the emergence of gametic disequilibrium in adapted populations, as allelic combinations are selected for or against, resulting in deviations from Mendelian proportions between pairs of unlinked loci [@Boyrie2021-jo].

Identifying the signatures of polygenic and epistatic response to selection is a challenging problem, and Evolve and Resequence (E&R) experiments have emerged as a natural and powerful tool for investigating these questions [@Barghi2020-ie].
By exposing populations to a stressful treatment condition and keeping them in this environment for several generations, without any sort of phenotype tracking or artificial selection, we are effectively letting natural selection take over and tracking the resulting changes in genetic composition of the selected populations. One consequence of this is that the flies are also under natural selection due to laboratory conditions, which introduces selective pressures other than the experimental treatment. Some E&R experiments remedy this by starting with laboratory adapted populations [eg., @Turner2011-tc], but these populations could still be changing in response to lab conditions even after many generations, and even subtle changes in rearing due to the selection experiment conditions could lead to confounds. We approach this problem by tracking the allele frequency changes in the selected and control populations at several generations, not just at the beginning and at the end of the experiment. This allows for the separation of the genomic effects of the stressor from the effects of the general laboratory environment.


Here, we performed an E&R experiment where three replicate populations of _Drosophila melanogaster_ were exposed to a stressful environment in the form of a high level of dietary sugar, while another three replicate populations were maintained on a control diet. All six populations were derived from the same base population. Whole genome sequencing was performed on flies from all six populations at generations 1, 11, 25, and 100, giving a total of almost 3000 sequenced individuals. Using this time series genomic data, we identify two directions of allele frequency change. The largest driver was shared across selected and control populations, suggesting a shared lab environment selection. The second largest driver of genetic change contrasts control and selected populations, and so is linked to the selection regime. We estimate that at least 6% of the genome was under positive selection due to high sugar stress. Most observed changes in allele frequency are however relatively modest, and using the individual sequence data to estimate haplotypes, we show how most of the selected loci do not show archetypal signals of selective sweeps after 100 generations. These results point towards a highly polygenic selection response, in line with theoretical expectations from quantitative genetics theory. We also quantify correlations in allele frequency between pairs of selected loci over time, as well as gametic disequilibrium after 100 generations of adaptation, and show that several allele show correlations and gametic disequilibrium across unlinked loci, suggesting that epistatic interactions participated in the response to selection. We confirm this last point by using Wright-Fisher simulations, showing that the correlations and the gametic disequilibrium we observe are unlikely to appear in the absence of epistatic interactions. While we lack a clear phenotype to directly study the effect of epistatic variation on selection response, our results suggest that epistatic contributions to polygenic response to selection are common and lead to detectable genomic signatures.

# Methods


# Results

## Polygenic Selection Response

After quality control (see Methods), we obtained allele frequency estimates for ~1.76M SNPs, giving a total of 4 time points x 3 replicate populations x 2 treatments x 1.76M SNPs data points. To identify the main drivers of genetic change without any prior assumptions, we performed a Principal Component Analysis (PCA) of the allele frequencies across the entire selection experiment. The first two principal components (PC), explaining 17% and 13% of the variance respectively, largely coincides with time and selection regime ([@fig:pca; @fig:PCscores_pvals]). This unsupervised approach thus identified time and high sugar selection as the two main drivers of genetic change genome wide. Surprisingly, the time dimension, captured by the first PC, explained slightly more variance than the selection regime, which was captured by the second PC. This shows that all six populations experience some common selection pressures, presumably related to lab environment. The second largest driver of genetic change is indeed exposure to high sugar stress ([@fig:pca]).

![Principal components one (x-axis) and two (y-axis) from the PCA on the genome wide allele frequencies across the entire selection experiment. Each line corresponds to one of the six experimental populations, red indicating high sugar treatment and blue control. The symbols indicate the generation at which the population was sampled. The variance explained per principal component is shown in the inset. ](./figures/PCs12_inset_allSNPs.png){ #fig:pca }

## High sugar selection on individual loci

To look for individual loci under selection, we fitted a regression model per SNP incorporating allele frequencies across all time points, replicate populations, and selection regimes. This model identifies SNPs whose allele frequency changes in the same direction over time in all replicate populations. The time coefficient in the model captures changes that are similar across all six populations (@fig:regression **PANEL**), and the time-by-regime coefficient captures changes that are unique to one selection regime (@fig:regression C). The p-values of the time coefficient were highly correlated with SNP loadings onto PC1 (cor = 0.59, p < 10-16, @fig:PCscores_pvals **PANEL**), whereas the the p-values of the time-by-regime coefficient were highly correlated with SNP loadings onto PC2 (cor = 0.68, p < 10-16, @fig:PCscores_pvals **PANEL**), consistent with the first two PCs capturing time and selection regime.

Different SNPs showed very different allele frequency trajectories over time. Some respond similarly to selection in all replicate populations regardless of selection regime (@fig:regression A), while others respond in opposite directions (@fig:regression C) or in only one of the regimes (@fig:regression B). Our regression model allowed us to distinguish these different scenarios and, for what follows, we focus on the selection signatures that are unique to the high sugar selection regime. The Manhattan profile in @fig:pca D, showing the time-by-regime p-values, suggests a polygenic selection response. This is in line with the observation that time and selection regime are the two main drivers of genetic change genome-wide (@fig:pca). 

![ Estimates from the per SNP regression model. Each panel shows a possible pattern of relevant and consistent allele frequency change across the six populations. (A) Consistent change in C and HS (B) C only (C) HS and C differ. Both B and C lead to a significant interaction term between time and treatment, but we filter SNPs that behave like those in panel B and change only in Control; (D) Manhattan plot showing negative log10 transformed p-values from the regression analysis of allele frequency over time. The p-values correspond to the time-by-selection regime interaction coefficient in the model. A significant p-value thus indicates different trajectories in the two treatments. SNPs showing a selection response primarily in the control regime are not shown. ](./figures/manhattan_pInt_HSonly_cut1e-9.png){ #fig:regression }

In order to further relate the locus-specific results (@fig:regression) to the genome wide signal quantified by the PCA (@fig:pca), we repeat the PCA after excluding all SNPs with a regression p-value below a given threshold. Changing the significance threshold allowed us to evaluate the effects of the filtered SNPs on the PCA. When using a very conservative threshold, excluding only the most strongly selected SNPs, the results from the PCA remained largely unchanged, showing that the PCA signal is not driven by a few loci under very strong selection (@fig:pca_threshold **PANEL**). We used these changes in the PCA as an heuristic to pick a p-value threshold of $10^{-12}$, since PC2 did no longer distinguish the different selection regimes when excluding SNPs with a p-value below this threshold (@fig:pca_threshold **PANEL**). SNPs passing this significance threshold are thus driving the majority of the selection response to high sugar stress that we observe in the PCA.

### What proportion of the genome is responding to selection?

Using this conservative threshold, ~**45K SNPs** show a signature of positive selection that is unique to the high sugar selection regime. Considering 200bp around every selected SNP, corresponding to an average $r^2$ of 0.2 (**Figure 1D todo: WHAT IS THIS?**), these SNPs span ~8.6Mb, or ~6% of the mappable genome of _D. melanogaster_. Since the LD around the selected loci is expected to be larger than the genome-wide average, we believe this to be a conservative estimate. The magnitudes of the allele frequency changes tend to be relatively small. Comparing generation 1 to generation 100, the mean change across all SNPs in the populations exposed to the high sugar selection regime is 0.11 (@fig:delta_af), while the mean change among the selected SNPs is 0.25 (@fig:delta_af). Among all the 1.76M SNPs, only 4753 show a pattern where the minor allele at generation 1 has reached fixation at generation 100 in at least one of the populations in the high sugar selection regime. Furthermore, many SNPs also display a delayed selection response, with the largest change in allele frequency after generation 25 (@fig:delta_af). This is consistent with theoretical predictions for polygenic adaptation involving independent loci [@Pavlidis2012-ly; @Chevin2008-gx], but could also be due to epistatic effects [@Paixao2016-gf]. 

![Histograms showing mean changes in allele frequency in the populations exposed the high sugar selection regime, between generation 1 and 11 (pink), 1 and 25 (green), and 1 and 100 (blue). Panel A includes all 1.76M SNPs and panel B includes the 71K SNPs show a signature of positive selection that is unique to the high sugar selection regime. ](./figures/placeholder.jpg){ #fig:delta_af }

### Do the selected alleles show a detectable sweep signature?

Next, we ask: do the identified selection signatures tend to coincide with the genomic footprint of selective sweeps? To this end, we estimated individual haplotypes at generation 100, using a core set of 20K high confidence SNPs. The haplotypes were then used to calculate the integrated Haplotype Score (iHS) [@Voight2006-rr] in the high-sugar selected populations. A large iHS indicates an extended haplotype associated with one allele at a given SNP, a pattern characteristic of a selective sweep. The estimated iHS and the p-values from our regression model showed a small but significant correlation (cor = 0.07, p = 2.1*10-17, @fig:iHS), indicating a tendency of longer haplotypes at the selected loci. The observed correlation is however very modest, showing that by and large, the loci indicated to be under selection using our time series data do not display a sweep-like pattern after 100 generations of adaptation. At a nominal significance threshold of p < x, only y% of the selected loci, as inferred from the regression analysis, also displayed a significant iHS. Taken together, these observations are all consistent with the polygenic view of adaptation through subtle shifts in allele frequency at a large number of loci, and with selection acting primarily on standing genetic variation rather than novel mutations. 

## Does epistasis contribute to selection response?

# Discussion

Clear polygenic response, in several replicas. Clear albeit indirect evidence of epistatic interactions in the responding loci.

\footnotesize


# Methods

## Mapping Population

To allow the detection of allelic effects that would be hidden in natural populations due to low frequency, we created a synthetic outbred mapping population. To create this population, we selected 16 inbred lines from the Netherlands population from the [Global Diversity Lines]{.smallcaps} [@Grenier2015-ie]. The lines were selected based upon their low frequency of inversion to reduce the suppression of recombination associated with inversions [@Barghi2019-fh]. To establish the population from these lines, we performed a round-robin cross on the initial lines (1 \Female -> 2 \Male, 2 \Female -> 3 \Male, ..., 16 \Female -> 1 \Male) and subsequently performed a round-robin cross on the F1s to ensure parental representation and that no chromosome was lost. The resulting F2 individuals were placed in large cages and allowed to recombine freely for more than 50 generations. This design increases the allele frequency of rare variants by replicating and randomizing throughout the population.

## Selection Regime

We performed a laboratory natural selection experiment [@Fuller2005-wn] on high-sugar diets without selecting for any phenotype. High-sugar diets are known to have high fitness costs [@Musselman2011-zq; @Na2013-wq; @Pallares2020-qa] and by allowing our populations to directly evolve on this, we explore the adaptation to this deleterious effect. To do this, we subdivided our mapping population into 6 replicate populations, 3 of which were placed on a standard media and 3 of which were placed on high-sugar media. The standard media consists of 8% glucose, 8% yeast, 1.2% agar, 0.04% phosphoric acid, and 0.4% propionic acid. High-sugar media follows the same recipe as standard media with the addition of 12% glucose resulting in a total of 20% glucose. Each population is placed in a population cage (BugDorm #4F3030) and maintained at ~5000 individuals for ~120 generations. Each generation was seeded from an egg lay, on fresh bottles of the respective diet, at 5-6 days post-eclosion. After pupation but before eclosion, bottles were moved to new cages and opened. Following each egg lay, individuals were collected and stored at -80C for subsequent sequencing.

## Library preparation and sequencing

Flies from generation 1, 11, 25, and 100 were selected from each population for sequencing and plated in 96 well plates. One 2.8mm stainless steel grinding bead (OPS diagnostics, #089-5000-11) and 100µl of lysis buffer were added to each well. Flies were homogenized for 10 minutes at maximum speed in a Talboys High Throughput Homogenizer (#930145). The resulting lysate was moved to a new 96-well plate for DNA extraction, using a Multi-Well Plate Vacuum Manifold (Pall Life Sciences #5017) and Acroprep advance 1mL DNA binding plates (Pall Life Sciences #8132).

Library prep was performed using a liquid handling robot (CyBio FeliX, Analitik Jena) to ease the processing of many samples and reduce variability from manual handling of samples. The protocol broadly followed the strategy described in Picelli et al [@Picelli2014-ji]. Specifically, we added 10µl (100µM) of forward oligo adapter A and 10µl(100µM) of reverse oligo adapter (Tn5MERev) to 80µl of reassociation buffer (10mM Tris pH 8.0, 50mM NaCl, 1mM EDTA). Following this, we annealed in a thermocycler with the following program: 95°C for 10 minutes, 90°C for 1 minute, reduce the temperature by 1°C per cycle for 60 cycles, and then hold at 4°C. The process was repeated for oligo adapter B. To load the adapters onto Tn5, we mixed 5µl of Tn5, 9µl of pre-annealed adapter A, and 9µl of pre-annealed adapter B then incubated this mixture in a thermocycler at 37°C for 30 minutes. The resulting pre-charged tn5 was the diluted with a 1:1 solution of reassociation buffer and glycerol to 1:1 reassociation buffer:glycerol to pre-charged Tn5.

## Mapping of reads, SNP calling, and estimation of allele frequencies
<!-- https://github.com/Lufpa/DNAseq_pipeline -->

## Inference of patterns of polygenic adaptation using PCA

To explore the main drivers of genetic changes during the course of the experiment, we performed a principal component analysis of the genome wide allele frequencies. For this analysis, we first assembled a sample-by-SNP matrix P, containing the genome wide allele frequencies in all 24 population samples (4 time points x 2 treatments x 3 replicate populations). We then computed the principal components of this matrix, allowing us to identify the main drivers of allele frequency changes in an unsupervised fashion.

## Inference of individual loci selection signatures

To detect signals of positive selection, we fitted the following logistic regression model:

$$
log(\frac{p_i}{1-p_i}) = \beta_t*t_i + \beta_{HS}*{HS}_i + \beta_{HS*t}*t_i*{HS}_i + e_i
$$

where $p_i$ denotes the allele frequency at a given locus in a given population and time point; $t_i$ is a numerical variable corresponding to the 4 sampled time-points (generations 1, 11, 25, and 100 are numerically coded as {1, 2, 3, 4}, and so the corresponding coefficient ($\beta_t$) measures the average allele frequency change across all time-points); $HS_i$ is an indicator variable corresponding to the two treatments ($HS=1$ in high sugar and $HS=0$ in control); $e_i$ is an error term. The $\beta$ variables are the corresponding regression coefficients. This allowed us to model the allele frequency for every locus across the entire selection experiment in one joint statistical framework. We focus primarily on the interaction effect $\beta_{{HS}*t}$, which quantifies the degree to which the allele frequency trajectory in the control regime differs from the one in the high sugar regime.

After fitting this model for all SNPs, we obtained estimates of the effect of time separately for the control and high sugar selection regimes. This was done using the emtrends function in the R package emmeans [@emmeans]. In order to exclude selection signatures that did not correspond to high sugar adaptation, we disregarded SNPs where the effect of time in the high sugar selection regime showed a p-value above $10^{-4}$  .



## Code and data availability

Code to reproduce figures in this paper is available at [github.com/ayroles-lab/highsugar-selection-code](https://github.com/ayroles-lab/highsugar-selection-code).

Corresponding data is available at **Princeton DataSpace link**.

## Supporting information and figures

Supporting information can be found at [github.com/diogro/HighSugarSelection](https://github.com/diogro/HighSugarSelection).

![Comparison of PC scores and p-values from regression analysis. PC1 scores are similar to p-values from SNPs that show a shared selection signal across treatment and control, and PC2 scores are similar to p-values from SNPs that only show selection signal in HS.](./figures/placeholder.jpg){#fig:PCscores_pvals tag="S1"}

![Across population allele frequency PCA (as in @fig:pca) after exclusion of significant SNPs. As the p-value significance threshold for the high sugar selection interaction term is increased and more putatively selected SNPs are removed, PC2 gradually explains less and less of the divergence between selected and control populations.](./figures/placeholder.jpg){#fig:pca_threshold tag="S2"}

![Negative log10 transformed p-values corresponding to the time-by-selection regime interaction coefficient in the regression analysis of allele frequency over time (y-axis) plotted versus absolute iHS scores (x-axis). 15K SNPs included in the figure, with each point representing one SNP](./figures/placeholder.jpg){#fig:iHS tag="S3"}


# References


